feature/staticroute/static_route_support/README.md

```diff
@@ -0,0 +1,114 @@
+*   Configure one IPv6 destination i.e. ipv6-network connected to ATE
+
+### RT-1.26.1 [TODO: https://github.com/openconfig/featureprofiles/issues/2489]
+#### Test to validate static route ECMP
+
+*   Configure IPv4 static routes:
+    *   Configure one IPv4 static route i.e. ipv4-route-a on the DUT for destination ipv4-network with the next hop of port-1
+    *   Configure another IPv4 static route i.e. ipv4-route-b on the DUT for destination ipv4-network with the next hop of port-2
+*   Configure IPv6 static routes:
+    *   Configure one IPv6 static route i.e. ipv6-route-a on the DUT for destination ipv6-network with the next hop of port-1
+    *   Configure another IPv6 static route i.e. ipv6-route-b on the DUT for destination ipv6-network with the next hop of port-2
+*   Initiate traffic from ATE port-3 towards destination ipv4-network and ipv6-network
+*   Validate that traffic is received from DUT on both port-1 and port-2 and ECMP works
+
+### RT-1.26.2 [TODO: https://github.com/openconfig/featureprofiles/issues/2489]
+#### Test to validate static route metric
+
+*   Configure metric of ipv4-route-b and ipv6-route-b to 1000
+    *   /network-instances/network-instance/protocols/protocol/static-routes/static/next-hops/next-hop/config/metric
+*   Validate that the metric is set correctly
+    *   /network-instances/network-instance/protocols/protocol/static-routes/static/next-hops/next-hop/state/metric
+*   Initiate traffic from ATE port-3 towards destination ipv4-network and ipv6-network
+*   Validate that traffic is received from DUT on port-1 only
+
+### RT-1.26.3 [TODO: https://github.com/openconfig/featureprofiles/issues/2489]
+#### Test to validate static route prefrence 
+
+*   Configure preference of ipv4-route-a and ipv6-route-a to 1000
+    *   /network-instances/network-instance/protocols/protocol/static-routes/static/next-hops/next-hop/config/preference
+*   Validate that the preference is set correctly
+    *   /network-instances/network-instance/protocols/protocol/static-routes/static/next-hops/next-hop/state/preference
+*   Initiate traffic from ATE port-3 towards destination ipv4-network and ipv6-network
+*   Validate that traffic is now received from DUT on port-2 only
+
+### RT-1.26.4 [TODO: https://github.com/openconfig/featureprofiles/issues/2489]
+#### Test to validate static route tag
+
+*   Configure a tag of value 10 on ipv4-route-a and ipv6-route-a
+    *   /network-instances/network-instance/protocols/protocol/static-routes/static/config/set-tag
+*   Validate the tag is set
+    *   /network-instances/network-instance/protocols/protocol/static-routes/static/state/set-tag
+
+### RT-1.26.5 [TODO: https://github.com/openconfig/featureprofiles/issues/2489]
+#### Test to validate disabling of recursive next-hop resolution
+
+*   Remove the ipv4-route-[a|b] and ipv6-route-[a|b] static routes
+*   Configure ipv4 and ipv6 ISIS between ATE port-1 <-> DUT port-1 and ATE port-2 <-> DUT port2
```

Comment:
I think you need three ports?
I think you want to override an ISIS learned route by using a static route with an interface-ref set:

See: 
https://openconfig.net/projects/models/schemadocs/yangdoc/openconfig-network-instance.html#network-instances-network-instance-protocols-protocol-static-routes-static-next-hops-next-hop-config-recurse

and 
https://openconfig.net/projects/models/schemadocs/yangdoc/openconfig-network-instance.html#network-instances-network-instance-protocols-protocol-static-routes-static-next-hops-next-hop-interface-ref

Reply:
IMO this is correctly written for the validation of the recurse flag (if this is the objective)

Interface-ref is just one specific example of disabling the recursion (with an additional restriction), however, any IP next-hop can be resolved recursively (or not).

Reply:
> I think you want to override an ISIS learned route by using a static route with an interface-ref set:

No actually I don't want to advertise, ipv4-network and ipv6-network through IS-IS. Static route next hop i.e. ipv[4|6]-loopback will be advertised through ISIS and will be recursively resolved.
I will remove [line 70](https://github.com/openconfig/featureprofiles/blob/46788e97668c975a98719db28f9d7bed57febfe7/feature/staticroute/static_route_support/README.md?plain=1#L70)

Reply:
SGTM